Runtime/MeshOperations/MergeElements.cs

```diff
@@ -8,7 +8,7 @@ namespace UnityEngine.ProBuilder.MeshOperations
     /// <summary>
     /// Merging faces together.
     /// </summary>
-    static class MergeElements
+    public sealed class MergeElements
```

Comment:
This could use a more descriptive summary for the API documentation. We'll also want to have Amy (@unitALG ) review the existing documentation in this class now that it is becoming public.